docs/specs/xref.yml

```diff
@@ -554,104 +554,111 @@ outputs:
       }
     }
 ---
-# Markdown and SDP non-cyclic xref reference
+# circular reference within SDP UID definition a -> a
 inputs:
-  docfx.yml: |
-    template: {APP_BASE_PATH}data/template
-  docs/a.md: |
-    ---
-    title: Title from yaml header a
+  docfx.yml:
+  docs/a.yml: |
+    #YamlMime:TestData
     uid: a
-    ---
-    Link to @b?displayProperty=summary
-  docs/b.json: | 
+    xref: a
+  _themes/ContentTemplate/schemas/TestData.schema.json: |
     {
-      "$schema": "https://raw.githubusercontent.com/dotnet/docfx/v3/schemas/TestData.json",
-      "uid": "b",
-      "name": "name for b",
-      "description": "description for b",
-      "summary": "Link to @a?displayProperty=summary",
+      "properties": {
+          "uid": {
```

Comment:
```suggestion
          "uid": { "contentType": "uid" }
```

Reply:
think it is better to define string type here as well